xbmc/guilib/DirtyRegion.h

```diff
@@ -35,3 +35,10 @@ class CDirtyRegion : public CRect
 };
 
 typedef std::vector<CDirtyRegion> CDirtyRegionList;
+
+class CRenderingStats
+{
```

Comment:
GUIWindowManager.h isn't currently included by the GUI controls.

It is currently here as CDirtyRegionList is the other parameter to Process and the stats are comparable to the dirty region list (i.e. info on what will be rendered).
I can move it but adding a new include to each GUI control is somewhat painful so I'd like to sure it of the right place before doing it.